.github/CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+# https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+# Order is important; the last matching pattern takes the most precedence.
+
+*                                 @dutow @dAdAbird
+/contib/pg_tde/documentation/     @nastena1606
```

Comment:
@ImTheKai Anastasia has no write permissions in this repository. Could you please add her.

Reply:
Done